src/library/returnBook/ReturnBookUI.java

```diff
@@ -1,101 +1,100 @@
 package library.returnBook;
+
 import java.util.Scanner;
 
 
 public class ReturnBookUI {
```

Comment:
Please rename class to ReturnItemUI as per class diagram.

Reply:
https://github.com/elliotmitchum/ITC205/pull/42/commits/8b325666fce4f04d4977b7d4dd50069c5cf81a7c